src/hint.c

```diff
@@ -308,6 +308,7 @@ void ReadClientInfo(ClientNode *np, char alreadyMapped)
    if(np->minWidth == np->maxWidth && np->minHeight == np->maxHeight) {
       np->state.border &= ~BORDER_RESIZE;
       np->state.border &= ~BORDER_MAX;
+      np->state.border &= ~BORDER_MIN;
```

Comment:
Is there some reason to disallow minimize if an window doesn't wish to be resized?